py_gtfs_rt_ingestion/tests/test_batcher.py

```diff
@@ -39,7 +39,7 @@ def test_batch_class(capfd):
     assert out == f"Batch of {sum(files.values())} bytes in {len(files)} {config_type} files\n" 
 
     # Calling `trigger_lambda` method raises exception
-    with pytest.raises(NoImplException):
+    with pytest.raises(ArgumentException):
```

Comment:
again, not sure how to stub this.

probably one of those things that makes the most sense to catch on end to end testing.